src/components/Header.js

```diff
@@ -0,0 +1,66 @@
+import logoImg from "../assets/images/logo.png";
+import "./Header.css";
+
+// function LoginBtn() {
+//   <a className="login-button">로그인</a>;
+// }
+
+function Nav({ userInfo }) {
+  // 유저데이터가 있으면 유저데이터, 없으면 로그인 버튼
+  if (userInfo === null)
+    return (
+      <div className="nav">
+        <a href="/">
+          <img className="logo" src={logoImg} alt="logo" />
+        </a>
+        <a href="/signin.html" className="login-link">
+          로그인
+        </a>
+      </div>
+    );
+
+  const { email, profileImageSource } = userInfo;
+  return (
+    <div className="nav">
+      <img className="logo" src={logoImg} alt="logo" />
+      <div className="user-info">
+        <img
+          className="user-profile"
+          src={profileImageSource}
+          alt="user-profile"
+        />
+        <span className="user-email">{email}</span>
+      </div>
+    </div>
+  );
+}
+
+function FolderInfo({ folderInfo }) {
+  if (folderInfo === null) return;
+
+  const { name: folderName } = folderInfo;
+  const { name, profileImageSource } = folderInfo.owner;
+
+  return (
+    <div className="folder-info">
+      <img
+        className="folder-profile"
+        src={profileImageSource}
+        alt={`user folder profile`}
```

Comment:
단순히, string만 전달하는 부분이고 다른 컴포넌트에서와 동일한 컨벤션 유지를 위해서 아래처럼 수정해주시면 더 좋을 것 같습니다. (약간이지만 더 간단한 방식이기도 하구요!) 
```suggestion
        alt="user folder profile"
```